app/src/main/res/values/component_colors.xml

```diff
@@ -172,10 +176,7 @@
   <color name="component_color_license_text_viewer_activity_background_color">@color/color_palette_dark_background_color</color>
   <color name="component_color_license_text_viewer_activity_text_color">@color/color_palette_description_text_color</color>
   <!-- Onboarding Activity -->
-  <color name="component_color_onboarding_activity_background_color">@color/color_palette_light_background_color</color>
   <color name="component_color_onboarding_activity_sub_text_color">@color/color_palette_primary_text_color</color>
-  <color name="component_color_onboarding_activity_text_color">@color/color_palette_highlighted_text_color</color>
-  <color name="component_color_onboarding_activity_divider_color">@color/color_palette_onboarding_divider_color</color>
   <color name="component_color_onboarding_activity_dots_color">@color/color_palette_dots_color</color>
   <color name="component_color_onboarding_activity_arrow_color">@color/color_palette_arrow_color</color>
```

Comment:
@rt4914  in dark mode [mock ](https://xd.adobe.com/view/c05e9343-60f6-4c11-84ac-c756b75b940f-950d/screen/5981ecd6-2a80-4d82-8715-3e92294d53b5/specs/) forward arrow color is `#3BD1C4` but there it is `#FFFFFF`

Reply:
Done. PTAL at commit directly.